suite-native/module-device-settings/src/components/DeviceInteractionScreenWrapper.tsx

```diff
@@ -18,6 +19,7 @@ export const DeviceInteractionScreenWrapper = ({
     const device = useSelector(selectSelectedDevice);
 
     const closeAction = useCallback(() => {
+        TrezorConnect.cancel();
```

Comment:
This was removed in the previous commit and the second commit put it back, which I believe is correct.

Reply:
Yeah, the initial idea was to call `TrezorConnect.cancel `in `useEffect` on a specific screen when the user leaves the page.

```
useEffect(() => () => {
  if (deviceConnected) TrezorConnect.cancel();
}, []);
```

or something similar. But then, I realized two reasons why this wouldn’t be a good idea:

1. @yanascz  mentioned that he wants to reuse the `ContinueOnTrezor` page for his Bluetooth implementation as well, so I didn’t want to create obstacles for his implementation.

2.	If T`rezorConnect.cancel` were called twice (once in `useEffect` and once by pressing the `close` button), it caused issues where, if you tried to change the PIN or trigger another device setting action right after, it wouldn’t get triggered.